### PR TITLE
Added example how to use PrepareArbitraryStateCP

### PR DESCRIPTION
**Pull request by RufusJWB**

### CODE REVIEW

File: Standard/src/Preparation/Arbitrary.qs [hunk 1]
```diff
@@ -51,6 +51,15 @@ namespace Microsoft.Quantum.Preparation {
     /// elements $(r_j, t_j) = (0.0, 0.0)$ if fewer than $2^n$ are
     /// specified.
     ///
+    /// # Example
+    /// The following snippet prepares the quantum state $\ket{\psi}=\sqrt{1/8}\ket{0}+\sqrt{7/8}\ket{2}$
+    /// in the qubit register `qubitsLE`.
+    /// ```qsharp
+    /// use qubits = Qubit[1];
+    /// let qubitsLE = LittleEndian(qubits);
```
Comment (msoeken) [inline Standard/src/Preparation/Arbitrary.qs:59]:
```suggestion
    /// use qubits = Qubit();
    /// let qubitsLE = LittleEndian([qubits]);
```

File: Standard/src/Preparation/Arbitrary.qs [hunk 2]
```diff
@@ -51,6 +51,15 @@ namespace Microsoft.Quantum.Preparation {
...
+    /// PrepareArbitraryStateCP([ComplexPolar(1.0/Sqrt(2.0),0.0),ComplexPolar(1.0/Sqrt(2.0),PI()/2.0)],qubitsLE); // = |i>
```
Comment (msoeken) [inline Standard/src/Preparation/Arbitrary.qs:60]: What is `|i>`?

**Review by msoeken — CHANGES_REQUESTED** [review]
Thanks for your contribution. I made some small comments.

File: Standard/src/Preparation/Arbitrary.qs (hunk 2 above)
Comment (RufusJWB) [reply Standard/src/Preparation/Arbitrary.qs:60]: One of the common labels for labeling an axis of the Bloch sphere: https://stem.mitre.org/quantum/quantum-concepts/bloch-sphere.html#labeling-the-axis-points-of-the-bloch-sphere . We could also remove this here. It's just a little remark.

Comment (RufusJWB) [reply Standard/src/Preparation/Arbitrary.qs:60]: @msoeken Do you have more questions? Should I remove the |i> to get this PR accepted?

**Review by msoeken — APPROVED** [review]

**Pull request merged**